kuksa-client/kuksa_client/grpc/aio.py

```diff
@@ -251,19 +253,22 @@ async def set(self, *, updates: Collection[EntryUpdate], **rpc_kwargs) -> None:
                 grpc.*MultiCallable kwargs e.g. timeout, metadata, credentials.
         """
         paths_with_required_type = self._get_paths_with_required_type(updates)
+        
         paths_without_type = [
             path for path, data_type in paths_with_required_type.items() if data_type is DataType.UNSPECIFIED
         ]
         paths_with_required_type.update(await self.get_value_types(paths_without_type, **rpc_kwargs))
         req = self._prepare_set_request(updates, paths_with_required_type)
+        rpc_kwargs.__setitem__("metadata", self.authorization_header)
         try:
             resp = await self.client_stub.Set(req, **rpc_kwargs)
         except AioRpcError as exc:
             raise VSSClientError.from_grpc_error(exc) from exc
         self._process_set_response(resp)
 
-    async def authorize(self, *, token: str) -> str:
-        raise NotImplementedError('"authorize" is not yet implemented')
+    async def authorize(self, *, token: Dict) -> None:
+        token = "Bearer " + token
+        self.authorization_header.append(("authorization", token))
```

Comment:
This is not right.
It will keep appending more and more tuples to the list when authorize is called more than once.